tracpro/baseline/forms.py

```diff
@@ -0,0 +1,55 @@
+from django import forms
+
+from tracpro.polls.models import Poll
+from .models import BaselineTerm
+
+
+class BaselineTermForm(forms.ModelForm):
+    """
+    Form for Baseline Term
+    """
+    class Meta:
+        model = BaselineTerm
+        fields = ('name', 'org', 'region', 'start_date', 'end_date',
+                  'baseline_poll', 'baseline_question',
+                  'follow_up_poll', 'follow_up_question')
+
+        widgets = {
+            'start_date': forms.widgets.DateInput(attrs={'class': 'datepicker'}),
+            'end_date': forms.widgets.DateInput(attrs={'class': 'datepicker'}),
+            'org': forms.HiddenInput()
+        }
+
+    def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop('user')
+        org = self.user.get_org()
+
+        super(BaselineTermForm, self).__init__(*args, **kwargs)
+
+        if org:
+            self.fields['region'].queryset = self.user.get_regions(
+                org).order_by('name')
+            polls = Poll.get_all(org).order_by('name')
+            self.fields['baseline_poll'].queryset = polls
+            self.fields['follow_up_poll'].queryset = polls
+
+    def clean(self, *args, **kwargs):
+        cleaned_data = super(BaselineTermForm, self).clean()
+
+        start_date = cleaned_data.get("start_date")
+        end_date = cleaned_data.get("end_date")
+
+        if start_date > end_date:
```

Comment:
It is possible that one or both of `start_date` or `end_date` could be `None`. `None > None` works but you get a `TypeError` if you try to compare a `datetime` with None.

You can change the check to `start_date and end_date and start_date > end_date` to cover the error condition while allowing the default `This field is required` messages to suffice if one or both fields is `None`.